### ISSUE TITLE
[Feature Request] Migrate from RDBMS -> NoSQL

### ISSUE THREAD

**Issue by erickyun** [hyPnOtICDo0g/rss-chan#14]
Hi,

Thank you for your great project. 
İt would be great if you could add Mongodb for DATABASE_URL

**Comment by hyPnOtICDo0g** [comment]
@erickyun 
Yeah I've plans to switch to NoSQL, but I've just been busy. Someday hopefully I'll migrate for sure.

**Comment by erickyun** [comment]
> @erickyun Yeah I've plans to switch to NoSQL, but I've just been busy. Someday hopefully I'll migrate for sure.

Thank you for your response. I'm looking forward to it. 🙂👍

**Comment by LoveThisWorld002** [comment]
Any eta sir?
